examples/etc/containerd/firecracker-runtime.json

```diff
@@ -5,7 +5,5 @@
   "root_drive": "/var/lib/firecracker-containerd/runtime/default-rootfs.img",
```

Comment:
Since most customers wouldn't generate the configuration file on-the-fly, I'd like to keep this JSON file inside examples/.